bloom-housing/bloom
# chore: update translations and sign in link for lottery email

This PR addresses https://github.com/metrotranscom/doorway/issues/959 & https://github.com/metrotranscom/doorway/issues/958


## Description

The lottery published email for applicants has updated "what happens next" content, and the sign-in button link goes to the Sign In page instead of the homepage.

## How Can This Be Tested/Reviewed?

Run the updatesWhatHappensInLotteryEmail script runner endpoint and verify translations are updated.
Not sure the easiest way to test the email, but I added an automated test for that email template.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
